.emacs.d/settings/custom.el

```diff
@@ -8,7 +8,7 @@
 
 ; color theme
 (add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
-(load-theme 'solarized-dark 1)
+(load-theme 'solarized 1)
 (setq solarized-termcolors 256)
```

Comment:
I don't think so, this is what I have: 
```
;; color theme
(add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
(load-theme 'solarized t)
(set-frame-parameter nil 'background-mode 'dark)
(enable-theme 'solarized)
(setq solarized-termcolors 256)
```

I think, because of the recent changes, you will need to explicitly state the theme settings.